cristy-codes/calendars
# View other user's public calendars

Lists public calendars from other users.

Test:

1. Login as Bob, password bob
2. Toggle calendar privacy (9)
3. Logout (10)
4. Login as Ann, password ann
5. View calendars (8)
6. Notice you can see Bob's public calendar
